.travis.yml

```diff
@@ -1,10 +1,46 @@
-services:
-- docker
-script:
-- docker build --target test --tag todo-app:test .
-- docker run --mount type=bind,source="$(pwd)"/,target=/app/ todo-app:test tests/test_view_model.py
-- docker run --mount type=bind,source="$(pwd)"/,target=/app/ todo-app:test tests/test_endpoints.py
-- docker run  -eT_KEY=$T_KEY -eT_TOKEN=$T_TOKEN --mount type=bind,source="$(pwd)"/,target=/app/ todo-app:test tests_e2e
+services: docker
+jobs:
+ include:
+  - stage: Build docker Image
```

Comment:
This is more specifically building the test image. Your deploy stage builds the production image.